webapp/channels/src/components/user_settings/notifications/user_settings_notifications.tsx

```diff
@@ -1316,7 +1316,6 @@ class NotificationsTab extends React.PureComponent<Props, State> {
                     <div className='divider-dark'/>
                 </div>
             </div>
-
```

Comment:
Probably not intentional and should be reverted.

Reply:
Done ✅